glusterd2/utils/pprof.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import (
+	"net/http"
+	"net/http/pprof"
+	runtime_pprof "runtime/pprof"
+
+	"github.com/gorilla/mux"
+	log "github.com/sirupsen/logrus"
+)
+
+var (
+	basePath = "/debug/pprof/"
+)
+
+func addPath(router *mux.Router, name string, handler http.Handler) {
+	router.Path(basePath + name).Name(name).Handler(handler)
+	log.WithFields(log.Fields{
+		"name":    name,
+		"handler": handler,
+	}).Debug("Starting profile for %s", name)
```

Comment:
the name is logged twice in this log message.
once can be removed

Reply:
done